R/broom_wrapper.R

```diff
@@ -450,6 +454,95 @@ prediction <- function(df, data = "training", data_frame = NULL, conf_int = 0.95
   dplyr::group_by(ret, !!!rlang::syms(grouping_cols))
 }
 
+#' prediction wrapper for both training and test data
+#' @param df Data frame to predict. This should have model column.
+#' @export
+prediction_training_and_test <- function(df, prediction_type="default", threshold = 0.5, ...) {
+  test_index <- df %>% dplyr::filter(!is.null(.test_index)) %>% `[[`(1, ".test_index", 1)
+
+  grouped_cols <- colnames(df)[!colnames(df) %in% c("model", ".test_index", "source.data", ".model_metadata")]
+
+  tra_ret <- switch(prediction_type,
```

Comment:
Thought the variable name tra_ret is unnecessarily shortened. train_ret would be short enough and would not require extra imagination when reading.